### PR TITLE
6261/delete alert tests

### PR DESCRIPTION
**Pull request by gwyneplaine**
Minor changes to the test utils in the admin-ui-tests folder, and added tests for deletion success and failure alerts.

### CODE REVIEW

File: tests/admin-ui-tests/utils.ts [hunk 1]
```diff
@@ -115,7 +124,7 @@ export const adminUITests = (
       beforeAll(async () => {
         await deleteAllData(projectDir);
       });
-      tests(playwright[browserName]);
+      tests(playwright[browserName], deleteAllData);
```
Comment (gwyneplaine) [inline tests/admin-ui-tests/utils.ts:127]: pass `deleteAllData` to our test fn so we can call it as necessary from within each test file.

File: tests/admin-ui-tests/list-view-crud.test.ts [hunk 2]
```diff
@@ -0,0 +1,126 @@
+import { Browser, Page } from 'playwright';
+import fetch from 'node-fetch';
+import { adminUITests } from './utils';
+
+adminUITests('./tests/test-projects/crud-notifications', (browserType, deleteAllData) => {
+  let browser: Browser = undefined as any;
+  let page: Page = undefined as any;
+  const seedData = async (page: Page, query: string, variables?: Record<string, any>) => {
+    try {
+      const { errors } = await fetch('http://localhost:3000/api/graphql', {
```
Comment (gwyneplaine) [inline tests/admin-ui-tests/list-view-crud.test.ts:10]: we were doing things like this previously 
```
await page.evaluate((query, variable) => {
   return fetch('http://localhost:3000/api/graphql', {
        method: 'POST',
        headers: {
          'Content-Type': 'application/json',
        },
        body: JSON.stringify({
          query,
          variables,
        }),
      }).then(res => res.json());
}, { query, variable)
```
but I ripped that out in favor of node-fetch, as we were getting a few rather non-deterministic network issues with this method, (most likely due to shifting network contexts from where page.evaluate is called) 

Running requests to api from node seems like the better more deterministic choice here, but I'm happy to re-evaluate this choice if i'm missing something here.

File: yarn.lock [hunk 3]
```diff
@@ -4025,11 +4025,6 @@ babel-plugin-polyfill-regenerator@^0.2.2:
   dependencies:
     "@babel/helper-define-polyfill-provider" "^0.2.2"
 
-babel-plugin-remove-graphql-queries@2.7.2:
```
Comment (timleslie) [inline yarn.lock:4028]: Is this removal intentional?

Comment (gwyneplaine) [reply yarn.lock:4028]: definitely not, let me revert from master and see what's going on

**Review by timleslie — APPROVED** [review]
👍
